core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -2025,6 +2025,15 @@ package object config {
       .checkValue(v => v >= 0, "The threshold should be non-negative.")
       .createWithDefault(1L * 1024 * 1024)
 
+    private[spark] val CLEAN_BROADCAST_AFTER_EXECUTION_END_ENABLED =
+      ConfigBuilder("spark.broadcast.cleanAfterExecutionEnd.enabled")
+        .doc("Whether to enable clean broadcast data after sql execution. If enabled, " +
```

Comment:
nit:

```suggestion
        .doc("Whether to enable clean broadcast data after SQL execution. If enabled, " +
```

BTW, SQL configurations should go to `SQLConf`. If this is a static conf, it should go to `StaticSQLConf`

Reply:
@HyukjinKwon Thank you for your advice. But this parameter needs to be used in `BroadcastManager#initialize` (core module), but the core module does not import the SQL module. The core module use the parameters of the SQL module. I don’t know if they are appropriate.